apiCalls.js

```diff
@@ -92,16 +92,17 @@ export const addDogForUser = async (user_id, name, sex, breed, size, age, fixed,
   return newDog
 }
 
-export const patchUserPhoto = async (photo, id) => {
+export const patchUserPhoto = async (image, id) => {
   const options = {
     method: 'PATCH',
-    body: JSON.stringify({ photo }),
+    body: JSON.stringify({ image }),
     headers: {
       'Content-Type': 'application/json'
     }
   };
 
-  const resp = await fetch(`https://node-pupdates-backend.herokuapp.com/api/v1/users/${id}`, options)
+  const resp = await fetch(`https://backend-pupdates.herokuapp.com/api/v1/users/${id}`, options)
```

Comment:
So is this PATCH not successfully coming back from the backend-pupdates api?

Reply:
When I console log data on line 107, it shows the user data with the newly patched image but it isnt actually manipulating the user on the back end.

Reply:
Okay we'll have to discuss with Andrew!